{
  "ID": "bb65a78a-9f3f-4cfe-8d6f-faffeb7ab328",
  "Root": {
    "alive": true,
    "content": [
      "d941ab68-690d-4592-b981-66906c27576b",
      "fbbb3e3e-14f0-46bb-a465-fcae1ec80137",
      "d419c4d3-10b4-42c4-ae2c-a9234bba10fc"
    ],
    "created_by": "bb760e2d-d679-4b64-b2a9-03005b21870a",
    "created_time": 1552101277637,
    "format": {
      "page_full_width": true,
      "page_small_text": true
    },
    "id": "bb65a78a-9f3f-4cfe-8d6f-faffeb7ab328",
    "ignore_block_count": true,
    "last_edited_by": "bb760e2d-d679-4b64-b2a9-03005b21870a",
    "last_edited_time": 1552101277637,
    "parent_id": "d43d4630-d5f4-4667-982d-03e02440c201",
    "parent_table": "block",
    "properties": {
      "title": [
        [
          "Exceptions"
        ]
      ]
    },
    "type": "page",
    "version": 3,
    "content_resolved": [
      {
        "alive": true,
        "created_by": "bb760e2d-d679-4b64-b2a9-03005b21870a",
        "created_time": 1552101277636,
        "id": "d941ab68-690d-4592-b981-66906c27576b",
        "ignore_block_count": true,
        "last_edited_by": "bb760e2d-d679-4b64-b2a9-03005b21870a",
        "last_edited_time": 1552101277636,
        "parent_id": "bb65a78a-9f3f-4cfe-8d6f-faffeb7ab328",
        "parent_table": "block",
        "properties": {
          "title": [
            [
              "Remarks"
            ]
          ]
        },
        "type": "sub_header",
        "version": 1,
        "inline_content": [
          {
            "Text": "Remarks"
          }
        ]
      },
      {
        "alive": true,
        "created_by": "bb760e2d-d679-4b64-b2a9-03005b21870a",
        "created_time": 1552101277637,
        "id": "fbbb3e3e-14f0-46bb-a465-fcae1ec80137",
        "ignore_block_count": true,
        "last_edited_by": "bb760e2d-d679-4b64-b2a9-03005b21870a",
        "last_edited_time": 1552101277637,
        "parent_id": "bb65a78a-9f3f-4cfe-8d6f-faffeb7ab328",
        "parent_table": "block",
        "properties": {
          "title": [
            [
              "An "
            ],
            [
              "exception",
              [
                [
                  "i"
                ]
              ]
            ],
            [
              " is an object that represents the occurrence of an exceptional condition. In other words, it indicates that something went wrong."
            ]
          ]
        },
        "type": "text",
        "version": 1,
        "inline_content": [
          {
            "Text": "An "
          },
          {
            "Text": "exception",
            "AttrFlags": 4
          },
          {
            "Text": " is an object that represents the occurrence of an exceptional condition. In other words, it indicates that something went wrong."
          }
        ]
      },
      {
        "alive": true,
        "created_by": "bb760e2d-d679-4b64-b2a9-03005b21870a",
        "created_time": 1552101277637,
        "id": "d419c4d3-10b4-42c4-ae2c-a9234bba10fc",
        "ignore_block_count": true,
        "last_edited_by": "bb760e2d-d679-4b64-b2a9-03005b21870a",
        "last_edited_time": 1552101277637,
        "parent_id": "bb65a78a-9f3f-4cfe-8d6f-faffeb7ab328",
        "parent_table": "block",
        "properties": {
          "title": [
            [
              "In Ruby, "
            ],
            [
              "exceptions",
              [
                [
                  "i"
                ]
              ]
            ],
            [
              " are often referred to as "
            ],
            [
              "errors",
              [
                [
                  "i"
                ]
              ]
            ],
            [
              ". That’s because the base "
            ],
            [
              "Exception",
              [
                [
                  "c"
                ]
              ]
            ],
            [
              " class exists as a top-level exception object element, but user-defined execution exceptions are generally "
            ],
            [
              "StandardError",
              [
                [
                  "c"
                ]
              ]
            ],
            [
              " or descendants."
            ]
          ]
        },
        "type": "text",
        "version": 1,
        "inline_content": [
          {
            "Text": "In Ruby, "
          },
          {
            "Text": "exceptions",
            "AttrFlags": 4
          },
          {
            "Text": " are often referred to as "
          },
          {
            "Text": "errors",
            "AttrFlags": 4
          },
          {
            "Text": ". That’s because the base "
          },
          {
            "Text": "Exception",
            "AttrFlags": 2
          },
          {
            "Text": " class exists as a top-level exception object element, but user-defined execution exceptions are generally "
          },
          {
            "Text": "StandardError",
            "AttrFlags": 2
          },
          {
            "Text": " or descendants."
          }
        ]
      }
    ],
    "title": "Exceptions",
    "format_page": {
      "page_cover": "",
      "page_cover_position": 0,
      "page_font": "",
      "page_full_width": true,
      "page_icon": "",
      "page_small_text": true
    }
  },
  "Users": [
    {
      "email": "kkowalczyk@gmail.com",
      "family_name": "Kowalczyk",
      "given_name": "Krzysztof",
      "id": "bb760e2d-d679-4b64-b2a9-03005b21870a",
      "locale": "en",
      "mobile_onboarding_completed": true,
      "onboarding_completed": true,
      "profile_photo": "https://s3-us-west-2.amazonaws.com/public.notion-static.com/2dcaa66c-7674-4ff6-9924-601785b63561/head-bw-640x960.png",
      "time_zone": "America/Los_Angeles",
      "version": 19
    }
  ],
  "Tables": null
}